bokehjs/src/coffee/models/plots/plot_canvas.coffee

```diff
@@ -446,7 +438,7 @@ class PlotCanvasView extends Renderer.View
 
     for tool_view in tool_views
       level = tool_view.model.level
-      @levels[level][tool_view.model.id] = tool_view
+      @levels['tool'][tool_view.model.id] = tool_view
```

Comment:
Actually thinking back, I think there was point where some tools did their own drawing, and were in fact renderers. That's not currently the case anymore. If we want to dictate that tools may *have* renderers, but not *be* renderers, that's OK.